src/pages/academy/blog/index.js

```diff
@@ -112,6 +114,34 @@ export const query = graphql`
                 read_time_in_minutes
                 published_date
             }
+            blog_eu: blog(
+                filter: { status: { _eq: "published" }, hide_for_eu: { _eq: false } }
+                sort: "-published_date"
+            ) {
+                id
+                main_image {
+                    id
+                    description
+                    imageFile {
+                        childImageSharp {
+                            gatsbyImageData
+                        }
+                    }
+                }
+                slug
+                featured
+                hide_for_eu
+                tags {
+                    id
+                    tags_id {
+                        tag_name
+                    }
+                }
+                blog_title
+                blog_description
+                read_time_in_minutes
+                published_date
+            }
```

Comment:
For this page and also the videos page, we don't want to do a double query here as we are not limiting the query to a certain number. Instead, we can just declare `article_data` as `data.directus.blog.filter()` for the EU view.